DISCOVER/15_security_best_practices.md

```diff
@@ -0,0 +1,109 @@
+# Ensuring Privacy and Security in Event Management  
```

Comment:
As @aterrel wrote in a general comment, the title of this section is misleading as the focus here is on data systems, not physical privacy or security, which is also what I thought it would be initially.